eos/b-decays/b-to-gamma-l-nu_TEST.cc

```diff
@@ -0,0 +1,139 @@
+/* vim: set sw=4 sts=4 et foldmethod=syntax : */
+
+/*
+ * Copyright (c) 2022 Philip Lüghausen
+ *
+ * This file is part of the EOS project. EOS is free software;
+ * you can redistribute it and/or modify it under the terms of the GNU General
+ * Public License version 2, as published by the Free Software Foundation.
+ *
+ * EOS is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+#include <test/test.hh>
+#include <eos/b-decays/b-to-gamma-l-nu.hh>
+#include <eos/maths/integrate.hh>
+#include <eos/maths/integrate-impl.hh>
+#include <eos/observable.hh>
+
+using namespace test;
+using namespace eos;
+
+class BToGammaLeptonNeutrinoTest :
+    public TestCase
+{
+    public:
+        BToGammaLeptonNeutrinoTest() :
+            TestCase("b_to_gamma-l_nu_test")
+        {
+        }
+
+        virtual void run() const
+        {
+            static const double eps = 1e-6;
+
+            Parameters p = Parameters::Defaults();
+
+            // Values from [BBJW:2018A]
+            p["B_u::mu_0@FLvD2022"]  = 1.5;
+            p["B->gamma::mu@FLvD2022QCDF"]  = 1.5;
+            p["B->gamma::mu_h1@FLvD2022QCDF"]  = 4.7;
+            p["B->gamma::mu_h2@FLvD2022QCDF"]  = 4.5;
+            p["B->gamma::s_0@FLvD2022QCDF"] = 1.59;
+            p["B->gamma::M^2@FLvD2022QCDF"] = 1.35;
+            p["decay-constant::B_u"] = 0.192;
+            p["mass::B_u"] = 5.27929;
+            p["mass::b(MSbar)"] = 4.453796188717916; // fix m_b_pole@1-loop to 4.8
+            p["mass::rho^+"] = 0.77526;
+            p["B::lambda_E^2"] = 0.0625;
+            p["B::lambda_H^2"] = 0.125;
+            p["B::LambdaBar"] = 5.27929 - 4.246057986045;
```

Comment:
Where is the ``4.246057...``` coming from?

Reply:
That was not my question. Where is the value for the b-quark mass, 4.246..., coming from?

Reply:
The value is a remnant of how the code developed... there were changes for the renormalization scheme and the loop order. But the exact value does not matter for the sake of the test.